bin/internal/update_dart_sdk.ps1

```diff
@@ -22,13 +22,13 @@ $dartSdkLicense = "$cachePath\LICENSE.dart_sdk_archive.md"
 $engineStamp = "$cachePath\engine-dart-sdk.stamp"
 $engineRealm = (Get-Content "$flutterRoot\bin\internal\engine.realm")
 
-if (Test-Path "$flutterRoot\bin\internal\engine.version") {
-    $engineVersion = (Get-Content "$flutterRoot\bin\internal\engine.version")
+# Calculate the engine hash from tracked git files.
+if ($Env:LUCI_CONTEXT -eq "") {
+  $engineVersion = (git merge-base HEAD master)
```

Comment:
Is this dependent on the current working directory you invoke the script from?

Reply:
Nope:
PS /Users/codefu/src/flaux/engine/src/flutter> git merge-base HEAD master
1268b2d35bee0e7efa5d7d51bbc50604ccd21291

Reply:
I mean if you invoke `update_dart_sdk.ps1` from C:\.

Reply:
Because users will be invoking `flutter.bat` from all different working directories.

Reply:
Then no, that will not work and we can adjust it:

PS /Users/codefu/src> flutter --version
fatal: not a git repository (or any of the parent directories): .git

Reply:
in a sub-shell (does PS have these?) we could CD `$flutterRoot` first, then run `git merge-base ...`

Reply:
I'm experimenting with `Push-Location "$flutterRoot"` / Pop-Location

Reply:
Done. Updated both scripts.